src/Util/BuiltinOperations.php

```diff
@@ -364,11 +364,18 @@ public static function regexMatchFunc(...$args): bool
      */
     public static function ipMatch(string $ip1, string $ip2): bool
     {
-        $objIP1 = IP::parse($ip1);
+        if (str_contains($ip2, '/')) {
+            list($subnet, $mask) = explode('/', $ip2);
 
-        $objIP2 = Range::parse($ip2);
+            $ip = ip2long($ip1);
+            $subnet = ip2long($subnet);
+            $mask = 0xffffffff << (32 - intval($mask));
 
-        return $objIP2->contains($objIP1);
+            return ($ip & $mask) === ($subnet & $mask);
+        } else {
+
+            return $ip1 === $ip2;
+        }
```

Comment:
Can it work with ipv6 ?

Reply:
IPv6 support and addtional unit tests have been added this time.